git-hg/updateforesttags.sh

```diff
@@ -20,38 +20,39 @@
 # ===========================================================================
 
 if [ $# -lt 1 ]; then
-   echo Usage: $0 '[jdk8u|jdk9] (TAGS)'
+   echo Usage: "$0" '[jdk8u|jdk9] (TAGS)'
    echo Version supplied should match a repository in AdoptOpenJDK/openjdk-VERSION
    exit 1
 fi
-[ -z "$WORKSPACE" -o ! -d "$WORKSPACE" ] && echo Cannot access \$WORKSPACE: $WORKSPACE && exit 2
+[ -z "$WORKSPACE" ] || [ ! -d "$WORKSPACE" ] && echo Cannot access \$WORKSPACE: "$WORKSPACE" && exit 2
 
 MODULES=(corba langtools jaxp jaxws nashorn jdk hotspot)
 
 OPENJDK_VERSION="$1"
 shift
+# shellcheck disable=SC2124
 TAGS="$@"
 GITHUB_PROJECT=git@github.com:AdoptOpenJDK
 GITHUB_REPO="openjdk-$OPENJDK_VERSION"
 
-case $OPENJDK_VERSION in
+case "$OPENJDK_VERSION" in
    jdk8*) HG_REPO=http://hg.openjdk.java.net/jdk8u/jdk8u
           [ -z "$TAGS" ] && TAGS="jdk8u144-b34 jdk8u151-b12 jdk8u152-b16 jdk8u161-b12 jdk8u162-b12 jdk8u172-b03 jdk8u172-b11";;
    jdk9*) HG_REPO=http://hg.openjdk.java.net/jdk_updates/jdk9u
-          [ -z "$TAGS" ] && TAGS="jdk-9+181 jdk-9.0.1+11 jdk-9.0.3+9 jdk-9.0.4+11" ;;
+          [ -z "$TAGS" ] && TAGS="jdk-9+181 jdk-9.0.1+11 jdk-9.0.3+9 jdk-9.0.4+11";;
        *) Unknown JDK version - only jdk8u and jdk9 are supported; exit 1;;
 esac
 
 # Clean up
-rm    -rf $WORKSPACE/$GITHUB_REPO $WORKSPACE/openjdk
-mkdir -p  $WORKSPACE/$GITHUB_REPO $WORKSPACE/openjdk/mirror 
+rm    -rf "${WORKSPACE}"/"${GITHUB_REPO}" "${WORKSPACE}"/openjdk
```

Comment:
Perhaps change other instances to use `{}` (or remove them from this one) to be consistent?